src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -185,7 +179,9 @@ class BankAccountStep extends React.Component {
         const subStep = shouldReinitializePlaidLink ? CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID : this.props.achData.subStep;
         const plaidDesktopMessage = getPlaidDesktopMessage();
         const bankAccountRoute = `${CONFIG.EXPENSIFY.NEW_EXPENSIFY_URL}${ROUTES.BANK_ACCOUNT}`;
-        const error = lodashGet(this.props, 'reimbursementAccount.error', '');
+        const errors = lodashGet(this.props, 'reimbursementAccount.errors', {});
+        const errorMessage = _.isEmpty(errors) ? '' : _.last(_.values(errors));
```

Comment:
```suggestion
        const errorMessage = getLatestErrorMessage(this.props.reimbursementAccount);
```
https://github.com/Expensify/App/blob/main/src/libs/ErrorUtils.js#L39-L52